iron-form.html

```diff
@@ -394,6 +422,9 @@
     },
 
     _doFakeSubmitForValidation: function() {
+      if (!!Polymer.clientDoesNotSupportFormValidationUI)
```

Comment:
Consider saving a `!` and some extra chars by using the positive form (e.g., `clientSupportsFormValidationUI`)